hack/install-operator.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+# Copyright 2020 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# build and deploy the operator into Openshift using our quay account
```

Comment:
Build is not in this script, right? This is only about installation.

Reply:
Corrected.